client/ayon_sitesync/addon.py

```diff
@@ -8,15 +8,16 @@
 from collections import deque, defaultdict
 import click
 
-from .version import __version__
+import ayon_api
+
+from ayon_core.settings import get_studio_settings
+from ayon_core.modules import AYONAddon, ITrayModule, IPluginPaths
+from ayon_common.utils import get_local_site_id
```

Comment:
`ayon_common` is available only in AYON launcher process. This will crash in any other process.